files/default/neutron-ha-tool.py

```diff
@@ -233,6 +236,10 @@ def run(args):
     else:
         raise ValueError('Invalid agent_selection_mode')
 
+    if args.target_agent is not None:
```

Comment:
Comparison with None is very unusual (only in lxml and because a bug). `if args.target_agent:` is mostly expected, or `args.target_agent` can be 0, '' or something like that?